app/assets/stylesheets/footer.scss

```diff
@@ -20,6 +20,22 @@ footer.footer {
     font-weight: bold;
   }
 
+  ul#footer_powered {
```

Comment:
- Avoid using id selectors
- Avoid qualifying id selectors with an element.
- Selector `footer_powered` should be written in lowercase with hyphens